src/workflow/RunEnergyPlus.cpp

```diff
@@ -154,14 +154,30 @@ void OSWorkflow::runEnergyPlus() {
     // TODO: is this the right place /Do we want to do that if we chose epJSON?
     detailedTimeBlock("Saving IDF", [this, &inIDF] { workspace_->save(inIDF, true); });
 
+    std::ofstream stdout_ofs(openstudio::toString(runDirPath / "stdout-energyplus"), std::ofstream::trunc);
+
     // TODO: workflow-gem was manually running expandObjects prior to the potential serialization to json
     // Should we rather pass -x to the E+ cmd line?
     if (!workflowJSON.runOptions()->skipExpandObjects()) {
       const std::string cmd = fmt::format("\"{}\"", openstudio::toString(runDirResults.expandObjectsExe.native()));
       LOG(Info, "Running command '" << cmd << "'");
 
       int result = 0;
-      detailedTimeBlock("Running ExpandObjects", [&cmd, &result] { result = std::system(cmd.c_str()); });
+      detailedTimeBlock("Running ExpandObjects", [this, /*&cmd,*/ &result, &runDirPath, &runDirResults, &stdout_ofs] {
+        // result = std::system(cmd.c_str());
+        namespace bp = boost::process;
+        bp::ipstream is;
+        std::string line;
+        bp::child c(runDirResults.expandObjectsExe, bp::std_out > is);
+        while (c.running() && std::getline(is, line)) {
+          stdout_ofs << openstudio::ascii_trim_right(line) << '\n';  // Fix for windows...
+          if (m_show_stdout) {
+            fmt::print("{}\n", line);
+          }
+        }
+        c.wait();
+        result = c.exit_code();
+      });
```

Comment:
Swallow the ExpandObjects stdout, otherwise you'll still get these

```
ExpandObjects Started.
No expanded file generated.
ExpandObjects Finished. Time:     0.021
```